src/EnergyPlus/MoistureBalanceEMPDManager.cc

```diff
@@ -558,7 +558,7 @@ namespace MoistureBalanceEMPDManager {
 
         // Calculate slope of moisture sorption curve at current RH. [kg/kg-RH]
         dU_dRH = material.MoistACoeff * material.MoistBCoeff * pow(RHaver, material.MoistBCoeff - 1) +
-                 material.MoistCCoeff * material.MoistCCoeff * material.MoistDCoeff * pow(RHaver, material.MoistDCoeff - 1);
```

Comment:
I had to dig for the commit where this was added: https://github.com/NREL/EnergyPlus/commit/de255ee9b8dfa1050c7e99e702c56b82bd199ce3#diff-691a3b35c3d65b682c8cf7acda7ffac9R485

It was a commit by @nmerket.  I tag him here, not to code shame him (maybe a little...we gotta keep each other humble), but instead to make him aware of the change for two reasons:
 - Could you verify the change is correct, that the extra C coefficient **should** be removed?
 - If it is the correct fix, then the original commit message grabs my attention: "porting @jdwoods21 empd code from 7.1".  You might want to make sure the original code this was adapted from is also corrected with this change (if it's still being used at all).

Reply:
It looks correct to me. I'm going to have @jdwoods21 look at it as well. I think he should be able to make the final say.

Reply:
Yes, there should only be one material.MoistCCoeff term in that equation, not two. One should be deleted.

The original code was in Matlab, and doesn't have this issue. But I transported that to EnergyPlus Fortran a while back...it may have been introduced then.

Reply:
Thanks, we're good here then. Sorry about the bug. I deserve the code shame. I don't think anybody is using the old modified E+ 7.1 code anymore, so no need to backport.